.travis.yml

```diff
@@ -1,8 +1,6 @@
 ---
-before_install: gem update --system 2.1.11
+before_install: gem update --system 2.5.2
 script: "./script/tests"
 gemfile: "this/does/not/exist"
 rvm:
-  - 2.0
```

Comment:
we will still need to test this on MRI 2.0 since that ships with El Capitan, Yosemite and Sierra.

```
$ /usr/bin/ruby -v
ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin16]
```

Reply:
...which means we _might_ need to get more complex with our gemspec and build matrix to account for the version differences.

Reply:
But i started the Puppet 4 updates because i could not run Puppet 3.7.x on ruby 2.3.3 in High Sierra. Supporting Puppet 3 and 4 is not feasible because of the diffrent Hiera versions and slightly different syntax.

Reply:
Puppet 4 should work on MRI 2.0 (the [docs list 1.9 -> 2.1.x](https://puppet.com/docs/puppet/4.6/system_requirements.html#ruby) as supported). 

The idea would be that we have a test suite for 2.0, 2.1, 2.2 and 2.3 which will cover all the OS versions. We might even look to drop El Capitan now that High Sierra is out however I'll leave that for another time.